mypy.ini

```diff
@@ -43,9 +43,7 @@ files = src/sentry/analytics/,
         src/sentry/mail/notifications.py,
         src/sentry/models/debugfile.py,
         src/sentry/models/groupsubscription.py,
-        src/sentry/models/organizationoption.py,
-        src/sentry/models/projectoption.py,
-        src/sentry/models/useroption.py,
+        src/sentry/models/options/,
```

Comment:
Another potential location would be with the other options code in `sentry.options.models`. Although that options and these options serve very different purposes 🤔

Reply:
Yeah these option models are sort of catch-all settings and the options module is for options that act like environment variables.